docs/index.asciidoc

```diff
@@ -41,3 +41,44 @@ Notes:
 * the fluent uses a second-precision time for events, so you will never see
   subsecond precision on events processed by this codec.
 
+
+[id="plugins-{type}s-{plugin}-options"]
+==== Fluent Codec configuration options
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Setting |Input type|Required
+| <<plugins-{type}s-{plugin}-nanosecond_precision>> |<<boolean,boolean>>|No
+| <<plugins-{type}s-{plugin}-target>> |<<string,string>>|No
+|=======================================================================
+
+&nbsp;
```

Comment:
Do we need the `&nbsp;` ?

Reply:
not sure it's a copy :spaghetti:, @karenzone ?

Reply:
It's there for other plugins, so let's leave it, please.